src/mappings/oba-efo.sssom.tsv

```diff
@@ -35,164 +35,164 @@ EFO:0009287	interventricular septum thickness	measurement	skos:exactMatch	OBA:00
-EFO:0007581	blood manganese measurement	measurement	skos:exactMatch	OBA:2050061			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0007582	blood molybdenum measurement	measurement	skos:exactMatch	OBA:2050066			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0007583	blood nickel measurement	measurement	skos:exactMatch	OBA:2050064			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0007584	blood zinc measurement	measurement	skos:exactMatch	OBA:2050072			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0008282	serum amyloid A-1 protein measurement	measurement	skos:exactMatch	OBA:2050083			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0008283	serum amyloid P-component measurement	measurement	skos:exactMatch	OBA:2050090			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0009795	serum urea measurement	measurement	skos:exactMatch	OBA:2050065			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0010127	serum immunoglobulin measurement	measurement	skos:exactMatch	OBA:2050079			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0010952	urinary potassium measurement	measurement	skos:exactMatch	OBA:2050095			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0011012	blood bicarbonate measurement	measurement	skos:exactMatch	OBA:2050094			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0020736	serum paraoxonase/arylesterase 1 measurement	measurement	skos:exactMatch	OBA:2050098			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0600041	serum clozapine measurement	measurement	skos:exactMatch	OBA:2050086			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0600043	serum N-desmethylclozapine measurement	measurement	skos:exactMatch	OBA:2050093			HumanCurated	EFO	OBA	1	2023-01-19				orcid:0000-0001-8314-2140
-EFO:0005273	sleep depth	measurement	skos:broadMatch	OBA:1000905	sleeping behavior quality		HumanCurated	EFO	OBA	1	2023-02-03				orcid:0000-0001-8314-2140
-EFO:0004870	sleep measurement	measurement	skos:exactMatch	OBA:2040170			HumanCurated	EFO	OBA	1	2023-02-03				orcid:0000-0001-8314-2140
-EFO:0005272	sleep quality	measurement	skos:exactMatch	OBA:2040170			HumanCurated	EFO	OBA	1	2023-02-03				orcid:0000-0001-8314-2140
-EFO:0005271	sleep duration	measurement	skos:exactMatch	OBA:2040171			HumanCurated	EFO	OBA	1	2023-02-03				orcid:0000-0001-8314-2140
-EFO:0005280	sleep latency	measurement	skos:exactMatch	OBA:2040172			HumanCurated	EFO	OBA	1	2023-02-03				orcid:0000-0001-8314-2140
-EFO:0005274	sleep time	measurement	skos:exactMatch	OBA:2040173			HumanCurated	EFO	OBA	1	2023-02-03				orcid:0000-0001-8314-2140
+EFO:0004829	response to irinotecan	response to stimulus	skos:exactMatch	OBA:2040128	trait in response to irinotecan	process quality	HumanCurated	EFO	OBA	1	2022-06-29				orcid:0000-0001-8314-2140

```

Comment:
HumanCurated should be `semapv:ManualMappingCuration` everywhere, in all files. (this was the "old way")